prereqs.py

```diff
@@ -164,15 +162,14 @@ def check_prereqs(install=False):
                     print("Attempting to install wasm-pack")
                     subprocess.run(["sh", file_name], check=True)
 
-            wasm_pack_help = subprocess.check_output(["wasm-pack", "--help"])
-            wasm_pack_version = wasm_pack_help.decode().splitlines()[0]
+            wasm_pack_version = subprocess.check_output(["wasm-pack", "--version"])
         else:
             print(
                 "wasm-pack not found. Please install from https://rustwasm.github.io/wasm-pack/installer/"
             )
             exit(1)
 
-    version_match = re.search(r"wasm-pack (\d+)\.(\d+).\d+", wasm_pack_version)
+    version_match = re.search(r"wasm-pack (\d+)\.(\d+).\d+", wasm_pack_version.decode())
```

Comment:
You should update the `wasmpack_ver` tuple on line 21 if this required 0.12.1 now.

Reply:
Should it be updated though? 0.11.0 works, as does 0.12.1. It was just 0.12.0 that had the mistakenly removed `--version`.

Reply:
correct 0.11 and lower all work too. But I updated it to 0.12.1 now. 

I didn't want to arbitrarily decide on raising min versions myself, but it makes sense to periodically require latest anyway, and it's better this way than some special case check for 0.12.0 only.

Reply:
Right. With this change, if you have 0.12.0 it will throw an error. It would be weird to allow 0.11.0, and 0.12.1, but error on 0.12.0. I think just requiring 0.12.1 or later is the right fix. Thanks!